### PR TITLE
Backfilling missing templates to gitHub

### PR DESCRIPTION
**Pull request by bushlack**

### CODE REVIEW

File: rightclick.io.hosting1.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "providerId": "RightClick.io",
```
Comment (arnoldblinn) [inline rightclick.io.hosting1.json]: Let's leave providerId as all lower case.  rightclick.io

File: wpengine.com.dxp.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+   "providerId":"WPEngine.com",
```
Comment (arnoldblinn) [inline wpengine.com.dxp.json]: Make lower case.

File: wpengine.com.dxp.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+   "providerName":"WPEngine.com",
```
Comment (arnoldblinn) [inline wpengine.com.dxp.json]: Is this really their display name?

File: myshopmatic.com.shopmatic.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  "records": [
+    {
+      "groupId": "REDIRECT",
+      "type": "CNAME",
+      "host": "www",
+      "data": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline myshopmatic.com.shopmatic.json]: I'm really confused about shopmatic.  What is the company, host, and service?

Naming conventions:

Each company is a Service Provider.  

They get a providerId and a providerName.  The providerId should be a host (e.g. foo.com) and the name should be what they want shown in the UX (Foo Incorporated).

They need at least one service, but can have multiple.  These are idenfied by the serviceId/serviceName.

So for foo.com, they might have "mail" and "hosting".

The template for mail should be:

foo.com.mail.json with providerId = foo.com and serviceId = mail

For hosting

foo.com.hosting.json 

The shopmatic templates do not follow this convention and need to be fixed.

File: myshoptiq.com.shopmatic.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  "records": [
+    {
+      "groupId": "REDIRECT",
+      "type": "CNAME",
+      "host": "www",
+      "data": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline myshoptiq.com.shopmatic.json]: See shopmatic comment above.

File: shopmakerapp.com.shopmatic.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  "records": [
+    {
+      "groupId": "REDIRECT",
+      "type": "CNAME",
+      "host": "www",
+      "data": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline shopmakerapp.com.shopmatic.json]: See shopmatic comment above

File: shopmatic.go-online.me.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "Configures go-online.me service",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline shopmatic.go-online.me.json]: See shopmatic comment above

File: shopmatic.myshopmatic.com.json [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "Configures myshopmatic.com service",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline shopmatic.myshopmatic.com.json]: See shopmatic comment above

File: shopmatic.myshoptiq.com.json [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "Configures myshoptiq.com service",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline shopmatic.myshoptiq.com.json]: See shopmatic comment above

File: shopmatic.shpmtc.club.json [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "Configures shpmtc.club service",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline shopmatic.shpmtc.club.json]: See shopmatic comment above

File: shopmatic.shopmakerapp.com.json [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "Configures shopmakerapp.com service",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```
Comment (arnoldblinn) [inline shopmatic.shopmakerapp.com.json]: See shompatic comment above. I would think this is shopmatic.com.shopmakerapp.json (or myshopmatic.com.shopmakerapp.json).  

providerId = shopmatic.com (or myshopmatic.com)
serviceId = shopmakerapp
file = <providerId>.<serviceId>.json

File: shopify.com.website2.json [hunk 12]
```diff
@@ -0,0 +1,26 @@
+{  
+   "providerId":"shopify.com",
+   "providerName":"Shopify",
+   "serviceId":"website2",
```
Comment (pawel-kow) [inline shopify.com.website2.json]: What is the difference to https://github.com/Domain-Connect/Templates/blob/master/shopify.com.website.json ?
Seems like the same template, just variables are named differently.

**Comment by bushlack** [comment]
Good feedback.  Closing this PR, and I'll submit another when the shopmatic changes are resolved.

**Pull request closed without merging**